docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,146 @@
+
+```bash
+slack triggers create --trigger-def "triggers/sample-trigger.json"      
+```
+
+The above command will create a trigger for the selected workspace. Make sure to select the workspace you want and that it is appended by `(dev)`. This will create a dev instance of your app. Once the trigger is successfully created, you should see an output like this:
+
+```bash
+⚡ Trigger created
+   Trigger ID:   [ID]
+   Trigger Type: shortcut
+   Trigger Name: Sample Trigger
+   URL: https://slack.com/shortcuts/[ID]/[Some ID]
+```
+
+The provided URL will be what you use to run your Trigger. Copy this URL and save it somewhere; you'll need it for later.
+
+---
+
+### Run your app {#run-your-app}
+
+Now that your app and Trigger are successfully created, let's try running it!
+
+```bash
+# install the required project dependencies
+pip install -r requirements.txt
+
+# start a local development server
+slack run
+```
+
+Executing `pip install -r requirements.txt` installs all the project requirements to your machine.
+
+Executing `slack run` starts a local development server, syncing changes to your workspace's development version of your app.
+
+You'll be prompted to select a workspace to install the app to&mdash;select the development instance of your workspace (you'll know it's the development version because the name has the string `(dev)` appended).
+
+> 💡 If you don't see the workspace you'd like to use in the list, you can `CTRL + C` out of the `slack run` command and run `slack auth login`. This will allow you to authenticate in your desired workspace to have it show up in the list for `slack run`.
+
+Once the app is successfully ran, you'll see output in your Terminal to indicate it's running, similar to what you would see with any other Bolt for Python app. You can search for the `⚡️ Bolt app is running! ⚡️` message to make sure that your app has successfully started up.
+
+### Trigger your app's workflow {#trigger-workflow}
+
+With your app running, access your workspace and paste the URL from the Trigger you created in the [previous step](/bolt-python/tutorial/getting-started-future#setup-trigger) into a message in a public channel.
+
+> 💡 App Triggers are automatically saved as a channel bookmark under "Workflows" for easy access.
+
+Send the message and click the "Run" button that appears. A modal will appear prompting you to enter information to greet someone in your Slack workspace. Fill out the requested information.
+
+![Hello World modal](https://slack.dev/bolt-js/assets/hello-world-modal.png "Hello World modal")
+
+Then, submit the form. In the specified channel submitted in the form, you should receive a message from the app tagging the submitted user. The message will also contain a randomly generated greeting and the message you wrote in the form.
+
+The full app flow can be seen here:
+![Hello World app](https://slack.dev/bolt-js/assets/hello-world-demo.gif "Hello World app")
+
+---
+
+### Limitations
```

Comment:
My logic behind moving this section to the bottom of the tutorial was to simplify the start of the tutorial, to make it as simple as possible to get started

Let me know if this is the approach we want to have for this and if you think this creates a feeling of simplicity?

Reply:
Ahh, I gotcha! I think it might be good to have the limitations upfront so devs are well-educated on what their apps can and can't do prior to jumping in so they can make an educated decision about what technology + app type to use!

Reply:
Done 💯